xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3770,8 +3770,8 @@ platform.dashboard.wiki.welcome=Welcome to your wiki
 platform.dashboard.wiki.personal.empty.edit=edit the dashboard section in your profile
 platform.dashboard.wiki.personal.empty=Your dashboard is currently empty. You can {0} to configure it. In the mean time, the default dashboard is displayed below.
 platform.dashboard.space=Dashboard for space {0}
-platform.dashboard.space.activity=Activity Stream for space {0}
-platform.dashboard.space.documents=Documents in space {0}
+platform.dashboard.gadget.activity.title=Activity Stream for {0}
+platform.dashboard.gadget.documents.title=Documents in {0}
```

Comment:
We should use 'Pages' instead of 'Documents'. It will be consistent with homepage's gadget, create/rename steps, etc.